### ISSUE TITLE
Support for keyboard navigation

### ISSUE THREAD

**Issue by jfbourgon** [Leaflet/Leaflet#646]
When focus is on the Map object, user should be able to pan using keyboard arrows and able to zoom in/out  using the +/- key.
